lib/internal/event_target.js

```diff
@@ -62,7 +62,7 @@ const kPropagationStopped = Symbol('propagationStopped');
 const isTrusted = () => false;
 
 class Event {
-  constructor(type, options) {
+  constructor(type, options = null) {
     if (arguments.length === 0)
       throw new ERR_MISSING_ARGS('type');
     if (options != null)
```

Comment:
This can now perform a strict equality check:
```suggestion
    if (options !== null)
```

Reply:
Why?

Reply:
(undefined is impossible anyway)?

Reply:
Adding the `= null` initializer to the `options` parameter [is mostly equivalent to doing the following](https://tc39.es/ecma262/#sec-runtime-semantics-keyedbindinginitialization) before the beginning of the function body:
```js
if (options === undefined) options = null;
```

I say mostly, because an initializer also affects the function’s `length` property, whereas an `if` check in the body doesn’t.